int32 current_angle1
int32 current_angle2
int32 toff
bool motor1_fail
bool motor2_fail
